modules/internal/ChapelArray.chpl

```diff
@@ -3263,7 +3263,9 @@ module ChapelArray {
       if !a.stridable && b.stridable then
         compilerError("cannot assign from a stridable domain to an unstridable domain without an explicit cast");
 
-    a._instance.dsiAssignDomain(b, lhsPrivate=false);
+    // distributions/robust/arithmetic/resizing/resizeFromOtherLocale.chpl
+    // fails if this uses b._instance instead of b._value
+    a._value.dsiAssignDomain(b, lhsPrivate=false);
```

Comment:
Hmm, interesting. I'm going to check out this branch and poke around to see if my mental model of _arrs is still correct.